windows/wintools_complete_install_verify/wintools_complete_install_verify.yml

```diff
@@ -17,44 +17,60 @@
         - include_tasks: ../setup/test_setup.yml
           vars:
             create_current_test_folder: true
-        - name: "Set update VMware tools by default"
+
+        - name: "Set update VMware Tools by default"
           ansible.builtin.set_fact:
             update_vmtools: true
           when: update_vmtools is undefined
-        - name: "Set not install from ESXi bundled VMware tools by default"
+        - name: "Set not install from ESXi bundled VMware Tools by default"
           ansible.builtin.set_fact:
             vmtools_esxi_bundled: false
           when: vmtools_esxi_bundled is undefined
 
-        # If VBS is enabled, will not install under development tools due to secureboot is enabled
+        # If VBS is enabled, will not install under development VMware Tools due to secureboot is enabled
         - block:
             - include_tasks: ../../common/vm_get_vbs_status.yml
             - include_tasks: ../../common/skip_test_case.yml
               vars:
-                skip_msg: "Skip test case due to under development VMware tools can not test on VM with VBS enabled."
+                skip_msg: "Skip test case due to under development VMware Tools can not test on VM with VBS enabled."
                 skip_reason: "Not Applicable"
               when:
                 - vm_vbs_enabled is defined and vm_vbs_enabled | bool
                 - is_development_tools is defined and is_development_tools | bool
           when: guest_os_ansible_architecture == "64-bit"
 
-        - include_tasks: ../../common/skip_test_case.yml
-          vars:
-            skip_msg: "Test case '{{ ansible_play_name }}' is skipped because update_vmtools is set to: {{ update_vmtools }}"
-            skip_reason: "Skipped"
-          when: not update_vmtools | bool
+        - name: "Handle update_vmtools is set to false situation"
+          block:
+            - include_tasks: ../../common/skip_test_case.yml
+              vars:
+                skip_msg: "Test case '{{ ansible_play_name }}' is skipped because VMware Tools is already installed and running, while 'update_vmtools' is set to: {{ update_vmtools }}"
+                skip_reason: "Skipped"
+              when: vmtools_is_running
+            - name: "Testing failed"
+              ansible.builtin.fail:
+                msg: "VMware Tools is not running when it's installed and 'update_vmtools' is set to false."
+              when: not vmtools_is_running
+          when:
+            - not update_vmtools | bool
+            - vmtools_is_installed
```

Comment:
I think `vmtools_is_installed` checking can be moved to line 52.
If `vmtools_is_running=true`, vmtools_is_installed is definitely true, and this test case will be ended in `skip_test_case.yml`.
Otherwise, we can check whether vmtools_is_installed is true and decide to fail the test case or not.

Reply:
This mainly for VMware Tools is installed situation, if it's running, then we can skip this test case, if it's not running, then this test case will fail.

Reply:
What about situation when update_vmtools=false and vmtools_is_installed=false?

Reply:
If vmtools_is_installed is false, no matter update_vmtools is set or not, this testing will be exectuted.

Reply:
Got it. I think we can describe this in test case description like:
If Windows guest OS doesn't have VMware Tools installed, this case will install VMware Tools.
If Windows guest OS has VMware Tools installed and update_vmtools is set to true, this case will reinstall VMware Tools.

Reply:
Added in description, thanks.